react/features/shared-video/reducer.native.js

```diff
@@ -6,13 +8,17 @@ import { SET_SHARED_VIDEO_STATUS } from './actionTypes';
  * Reduces the Redux actions of the feature features/shared-video.
  */
 ReducerRegistry.register('features/shared-video', (state = {}, action) => {
+    const { videoId, status, time, ownerId } = action;
+
     switch (action.type) {
     case SET_SHARED_VIDEO_STATUS:
         return {
             ...state,
-            status: action.status
+            videoId,
```

Comment:
This is the mobile reducer, why this change?

Reply:
I just used destructuring for the status that's it.